src/webview/direct-connect-form.ts

```diff
@@ -250,11 +251,3 @@ export function post(
 export function post(type: any, body: any): Promise<unknown> {
   return sendWebviewMessage(type, body);
 }
-
-/** Similar to {@link ConnectionType}, but only used for telemetry purposes. */
-export type FormConnectionType =
-  | "Apache Kafka"
-  | "Confluent Cloud"
-  | "Confluent Platform"
-  | "Other";
-export type SupportedAuthTypes = "None" | "Basic" | "API" | "SCRAM" | "OAuth" | "Kerberos";
```

Comment:
Moved to [`src/directConnections/types.ts`](https://github.com/confluentinc/vscode/blob/ea66d9a2a28fb4c8399ebde9de5c75264ee95b16/src/directConnections/types.ts)